pydantic/_internal/_generate_schema.py

```diff
@@ -1537,7 +1415,8 @@ def _typed_dict_schema(self, typed_dict_cls: Any, origin: Any) -> core_schema.Co
                         and field_name in field_docstrings
                     ):
                         field_info.description = field_docstrings[field_name]
-                    self._apply_field_title_generator_to_field_info(self._config_wrapper, field_info, field_name)
```

Comment:
This was actually called twice, here in the `_typed_dict_schema()` and later in `_common_field_schema()`.